src/aat/java/uk/gov/hmcts/ccd/datastore/tests/functional/elasticsearch/ElasticsearchCaseSearchTest.java

```diff
@@ -0,0 +1,157 @@
+package uk.gov.hmcts.ccd.datastore.tests.functional.elasticsearch;
+
+import static org.hamcrest.CoreMatchers.is;
+import static uk.gov.hmcts.ccd.datastore.tests.fixture.AATCaseBuilder.FullCase;
+import static uk.gov.hmcts.ccd.datastore.tests.fixture.AATCaseBuilder.FullCase.DATE_TIME;
+import static uk.gov.hmcts.ccd.datastore.tests.fixture.AATCaseBuilder.FullCase.TEXT;
+import static uk.gov.hmcts.ccd.datastore.tests.fixture.AATCaseType.State;
+
+import io.restassured.response.ValidatableResponse;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import uk.gov.hmcts.ccd.datastore.tests.AATHelper;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class ElasticsearchCaseSearchTest extends ElasticsearchBaseTest {
+
+    ElasticsearchCaseSearchTest(AATHelper aat) {
+        super(aat);
+    }
+
+    @BeforeAll
+    void setUp() {
+        assertElasticsearchEnabled();
+        importDefinition();
+        createCases();
+    }
+
+    @Nested
+    @DisplayName("Search updated case")
+    class SearchUpdatedCase {
+
+        @Test
+        @DisplayName("should return updated case on search")
+        void shouldReturnUpdatedCaseOnSearch() {
+            String jsonSearchRequest = "{"
+                + "  \"query\": {"
```

Comment:
we can remove this duplication by reusing query creational methods in the ElasticSearchQueries class as suggested above

Reply:
done